### PR TITLE
Af 202 move max exposure setter to mainregistry

### PR DESCRIPTION
**Pull request by Thomas-Smets**

### CODE REVIEW

File: src/MainRegistry.sol [hunk 1]
```diff
@@ -519,14 +582,15 @@ contract MainRegistry is IMainRegistry, MainRegistryGuardian {
      * @dev No need to check equality of length of arrays, since they are generated by the Account.
      */
     function getListOfValuesPerAsset(
+        address creditor,
```
Comment (j-vp) [inline src/MainRegistry.sol:585]: other getlistofvalues has bc, creditor, asset stuff. Best to be consistent in the order

File: src/MainRegistry.sol [hunk 2]
```diff
@@ -542,12 +606,13 @@ contract MainRegistry is IMainRegistry, MainRegistryGuardian {
         address[] calldata assetAddresses,
         uint256[] calldata assetIds,
         uint256[] calldata assetAmounts,
-        address baseCurrency
+        address baseCurrency,
```
Comment (j-vp) [inline src/MainRegistry.sol:609]: also here, bc and creditor before the asset stuff

File: src/MainRegistry.sol [hunk 3]
```diff
@@ -572,12 +637,13 @@ contract MainRegistry is IMainRegistry, MainRegistryGuardian {
...
+        address creditor
```
Comment (j-vp) [inline src/MainRegistry.sol:641]: same

File: src/MainRegistry.sol [hunk 4]
```diff
@@ -597,12 +663,13 @@ contract MainRegistry is IMainRegistry, MainRegistryGuardian {
...
```
Comment (j-vp) [inline src/MainRegistry.sol:667]: same

**Review by j-vp — COMMENTED** [review]
Not fully reviewed yet

File: src/MainRegistry.sol (hunk 1 above)
Comment (Thomas-Smets) [reply src/MainRegistry.sol:585]: Had to change it here for a stack to deep

File: src/MainRegistry.sol [hunk 5]
```diff
@@ -255,6 +258,22 @@ contract MainRegistry is IMainRegistry, MainRegistryGuardian {
...
+    /**
+     * @notice Checks for a token address and the corresponding Id if it is allowed.
+     * @param asset The contract address of the asset.
+     * @param assetId The Id of the asset.
+     * @return A boolean, indicating if the asset is allowed.
+     */
+    function isAllowed(address asset, uint256 assetId) external view returns (bool) {
+        address pricingModule = assetToAssetInformation[asset].pricingModule;
+
+        if (pricingModule == address(0)) {
```
Comment (j-vp) [inline src/MainRegistry.sol:270]:
```suggestion
        if (pricingModule == address(0)) return false;
        return IPricingModule(assetToAssetInformation[asset].pricingModule).isAllowed(asset, assetId);
```

**Review by j-vp — APPROVED** [review]
lgtm after small comment

**Pull request merged**